test/utils/simulate-key-input.ts

```diff
@@ -0,0 +1,16 @@
+import {simulate} from 'test-drive-react';
+
+const inputs = new WeakSet();
+
+export default function simulateKeyInput(
+    input: HTMLInputElement,
+    value: string
+) {
+    if (inputs.has(input)) {
+        input.value += value;
+    } else {
+        input.value = value;
```

Comment:
I've checked usage of `simulateKeyInput` and seems like you don't need this piece of functionality. Bonus for removing it is that `simulateKeyInput` would be more straightforward.